core/src/test/kotlin/org/stellar/anchor/sep31/Sep31ServiceTest.kt

```diff
@@ -1076,31 +1052,4 @@ class Sep31ServiceTest {
     val ex = assertThrows<SepValidationException> { sep31Service.updateFee() }
     assertEquals("Quote is missing the 'fee' field", ex.message)
   }
-
-  @ParameterizedTest
-  @MethodSource("generateGetClientNameTestConfig")
-  fun `test getClientName when`(
-    allowedClientDomains: List<String>,
-    isClientAttributionRequired: Boolean,
-    expectedClientName: String?,
-    shouldThrowExceptionWithInvalidInput: Boolean,
-  ) {
-    every { sep10Config.allowedClientDomains } returns allowedClientDomains
-    every { sep10Config.isClientAttributionRequired } returns isClientAttributionRequired
-    every { clientsConfig.getClientConfigBySigningKey(lobstrClientConfig.signingKey) } returns
-      lobstrClientConfig
-
-    // client name should be returned for valid input
-    val clientName = sep31Service.getClientName(lobstrClientConfig.signingKey)
-    assertEquals(expectedClientName, clientName)
-
-    // exception maybe thrown for invalid input
-    every { clientsConfig.getClientConfigBySigningKey("Invalid Public Key") } returns null
-    if (!shouldThrowExceptionWithInvalidInput) {
-      val clientNameNotFound = sep31Service.getClientName("Invalid Public Key")
-      assertNull(clientNameNotFound)
-    } else {
-      assertThrows<BadRequestException> { sep31Service.getClientName("Invalid Public Key") }
-    }
-  }
```

Comment:
Is this test removed? Or is it merge/rebase conflict?

Reply:
Added back.